README_PODMAN.md

```diff
@@ -23,26 +23,34 @@ should already exist.
 ## Configure a CNI network for Podman
 
 1. Create a new network using `podman network create`.  For example, `podman network create foobar` will suffice.
-2. Using your favorite editor, edit `/etc/cni/net.d/foobar.conflist` and add the following with the plugins stanza:
+
+The feature will be automatically enabled for newly created networks via podman network create. If you want to add this feature
+to an exisiting network add the needed lines to `/etc/cni/net.d/foobar.conflist` using your favorite editor.
 ```
+{
+   "cniVersion": "0.4.0",
+   "name": "foobar",
+   "plugins": [
+      ...
       {
-     "type": "dnsname",
-         "domainName": "podman.io"
+         "type": "dnsname",
+         "domainName": "dns.podman"
       }
-
+   ]
+}
 ```
-The following example [configuration file](example/cni-podman1.conflist) shows a usable example for Podman.
 
 ## Example: container name resolution
+**Note**: we use the --network foobar here. Also, in this test image, the nginx server will respond with
+*podman rulez* on an http request.
 
-1. sudo podman run -dt --name web --network foobar quay.io/libpod/alpine_nginx:latest
-    5139d65d22135e9ecab511559d863754550894a32285befd94dab231017048c2
+```console
+foo@bar:~$ sudo podman run -dt --name web --network foobar quay.io/libpod/alpine_nginx:latest
```

Comment:
Please do not add `foo@bar:~$`, this makes the line difficult to cut and paste.

Reply:
done